ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1045 @@
+                port_def.get("containerPort")
+                == constants.OCS_METRICS_EXPORTER_HTTPS_PORT
+            ):
+                return
+    raise AssertionError(
+        f"ocs-metrics-exporter pod does not declare port "
+        f"{constants.OCS_METRICS_EXPORTER_HTTPS_PORT} (HTTPS); "
+        f"containers={[c.get('name') for c in pod_obj.pod_data.get('spec', {}).get('containers', [])]}"
+    )
+
+
+def get_consumer_names_from_storage_consumers(namespace=None):
```

Ruff (0.15.21)

[warning] Unused function argument: `namespace`

(ARG001)